src/test/java/posters/pageobjects/pages/browsing/CategoryPage.java

```diff
@@ -32,9 +32,10 @@
 
     @Override
     @Step("ensure this is a category page")
-    public void isExpectedPage()
+    public CategoryPage isExpectedPage()
     {
         productOverview.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.